### PR TITLE
fix long lines _episodes/06-func.md #498

### PR DESCRIPTION
**Pull request by katkoler**
Hi, one of 10 PRs for #498

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Thanks, @katkoler!

What would you say about lining up the text that you moved to the new lines with the above lines?
I mean, change this:
```md
- "If a variable is not defined within the function it is used, 
Python looks for a definition before the function call"
```
to this:
```md
- "If a variable is not defined within the function it is used, 
   Python looks for a definition before the function call"
```

**Comment by katkoler** [comment]
That's a good point! I did it to some, but I'll double check the other ones for this as well.

File: _episodes/06-func.md [hunk 1]
```diff
@@ -18,12 +18,16 @@ keypoints:
...
 - "Put docstrings in functions to provide help for that function."
-- "Specify default values for parameters when defining a function using `name=value` in the parameter list."
-- "Parameters can be passed by matching based on name, by position, or by omitting them (in which case the default value is used)."
-- "Put code whose parameters change frequently in a function, then call it with different parameter values to customize its behavior."
+- "Specify default values for parameters when defining a function using `name=value` 
+   in the parameter list."
+- "Parameters can be passed by matching based on name, by position, 
+   or by omitting them (in which case the default value is used)."
+- "Put code whose parameters change frequently in a function, 
+then call it with different parameter values to customize its behavior."
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: one more line to fix

Comment (katkoler) [reply _episodes/06-func.md]: Good spot! Thanks :)

**Pull request merged**